src/main/java/de/kaleidox/e2uClaim/interfaces/Initializable.java

```diff
@@ -1,7 +1,5 @@
 package de.kaleidox.e2uClaim.interfaces;
 
-import java.io.IOException;
-
 public interface Initializable {
-    void init() throws IOException;
+    void init() throws Throwable;
```

Comment:
Why do you make this change? `init()` should only throw exceptions that may ever occurr during read operations. If you do not know how to handle any other exception, rethrow it as an underlying exception to a `RuntimeException` with a detailed message.